backend/templates/_ocsp.conf

```
@@ -0,0 +1,7 @@
+{% if certificate and certificate_id > 0 -%}
+{% if ocsp_stapling == 1 or ocsp_stapling == true %}
+  # OCSP Stapling
+  ssl_stapling on;
+  ssl_stapling_verify on;
```

```suggestion
  ssl_stapling on;
  ssl_stapling_verify on;
{% if certificate.provider == "letsencrypt" %}
  ssl_trusted_certificate /etc/letsencrypt/live/npm-{{ certificate_id }}/chain.pem;
{% else %}
  ssl_trusted_certificate /data/custom_ssl/npm-{{ certificate_id }}/fullchain.pem;
{% endif %}
```
Should [ssl_trusted_certificate](http://nginx.org/en/docs/http/ngx_http_ssl_module.html#ssl_trusted_certificate) be included as well?

From the nginx docs:

> For verification to work, the certificate of the server certificate issuer, the root certificate, and all intermediate certificates should be configured as trusted using the [ssl_trusted_certificate](http://nginx.org/en/docs/http/ngx_http_ssl_module.html#ssl_trusted_certificate) directive.

Please note that, in my suggestion, I'm unsure about the custom SSL chain 🙂